src/app/view-config-upgraders.js

```diff
@@ -420,8 +427,34 @@ export function upgradeFrom1_0_10(config) {
 export function upgradeFrom1_0_11(config) {
   const newConfig = cloneDeep(config);
 
+  const {
+    datasets,
+    coordinationSpace,
+  } = newConfig;
+
+  if (coordinationSpace.embeddingType) {
+    // This array may contain more embedding types than
+    // the cells.json file actually contains, but the tradeoff is that
+    // we do not have to load the cells.json file to double check what
+    // embedding types are actually present. CellsJsonAsObsEmbedding
+    // will just load the embedding as null if it is not present.
```

Comment:
So nothing broke here, this is more just for speed-up?  What made you notice this?  This is pretty deep.